### ISSUE TITLE
Feature Request: Add option to force __future__ import of absolute_imports

### ISSUE THREAD

**Issue by nickwilliams-eventbrite** [timothycrosley/isort#727]
We would like to force all of the Python files in our project (if and only if they contain other import statements) to contain `from __future__ import absolute_import` (or add `absolute_import` if it's missing in an already-existing `__future__` import). A new setting for isort to instruct it to add `absolute_import` whenever needed would be a great feature.

**Comment by bpedersen2** [comment]
While adding the future import with the -a  commandline flag is already possible, it  would be nice if there would be an equivalent option for configfiles to validate that new files also include the appropriate imports.

**Comment by timothycrosley** [comment]
This is now supported in develop and will be released with isort 5.2.0.

via CLI:

```bash
isort --add-import `from __future__ import absolute_import` --append-only
```

or via config:

```ini
add_imports=from __future__ import absolute_import
append_only=true
```

**Comment by timothycrosley** [comment]
This was just deployed with version 5.2.0 of isort. Thanks! 

~Timothy
